lib/experimental/script-modules.php

```diff
@@ -8,11 +8,10 @@
  * @param array $settings Array of determined settings for registering a block type.
  * @param array $metadata Metadata provided for registering a block type.
  */
-function gutenberg_filter_block_type_metadata_settings_register_modules( $settings, $metadata = null ) {
+function gutenberg_filter_block_type_metadata_settings_register_view_module( $settings, $metadata = null ) {
 	$module_fields = array(
 		// @todo remove viewModule support in Gutenberg >= 17.8 (replaced by viewScriptModule).
-		'viewModule'       => 'view_script_module_ids',
-		'viewScriptModule' => 'view_script_module_ids',
+		'viewModule' => 'view_script_module_ids',
```

Comment:
This handles the deprecated `viewModule` field. Core >= 6.5 handles `viewScriptModule`.